Functions/RIPA.Functions.UserProfile/RIPA.Functions.UserProfile.csproj

```diff
@@ -4,6 +4,11 @@
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
     <UserSecretsId>81bf2225-559e-46d9-af96-af8b4d216835</UserSecretsId>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
I'm not sure how or why this showed up, but you can remove it.

Reply:
I'm getting weird behavior when I remove this ItemGroup, could we review this item?

Reply:
Sure thing, I'll pull it locally and take a look as well.